cmd/sonarExecuteScan.go

```diff
@@ -122,9 +123,14 @@ func runSonar(config sonarExecuteScanOptions, client piperhttp.Downloader, runne
 	if len(config.Organization) > 0 {
 		sonar.addOption("sonar.organization=" + config.Organization)
 	}
-	if len(config.ProjectVersion) > 0 {
-		// handleArtifactVersion is reused from cmd/protecodeExecuteScan.go
-		sonar.addOption("sonar.projectVersion=" + handleArtifactVersion(config.ProjectVersion))
+	if len(config.Version) > 0 {
```

Comment:
should we align with https://github.com/SAP/jenkins-library/blob/master/cmd/whitesourceExecuteScan.go#L277?

-> also consider descriptor ...?

Reply:
As the build descriptor file is not required for the scan itself I would like to depend only on the `version` provided by `artifactPrepareVersion`.